ferminet/train.py

```diff
@@ -495,10 +516,10 @@ def opt_update(t, grad, params, opt_state):
         loss = stats['loss']
         aux_data = stats['aux']
       elif cfg.optim.optimizer == 'none':
-        data, pmove = mcmc_step(params, data, subkeys, mcmc_width)
+        data, pmove = mc_step(params, data, subkeys, mcmc_width)
         loss, aux_data = total_energy(params, data)
       else:
-        data, params, opt_state, loss, aux_data, pmove = step(
+        data, params, opt_state, loss, aux_data, pmove = mc_training_step(
```

Comment:
Is there a reason we need to change the name here? I think we can safely revert this.